services/ui-src/src/components/PortalTable.tsx

```diff
@@ -50,6 +55,191 @@ const defaultColumn = {
   disableGlobalFilter: true,
 };
 
+// Convenient type for casting JS objects to string-indexed accessible TS object
+type StringIndexedObject = { [key: string]: string[] };
+// String-accessible object with types for each "internalName" value (spa, waiver)
+const commonTypes: StringIndexedObject = {
+  waiver: [
+    "1915(b) Initial Waiver",
+    "1915(b) Waiver Amendment",
+    "1915(b) Waiver Renewal",
+    "1915(c) Appendix K Amendment",
+    "Temporary Extension",
+  ],
+  spa: ["Medicaid SPA", "CHIP SPA"],
+};
+/* TODO: This could be better handled by moving app-api/libs/status-lib.js to
+ *   the cmscommonlib, however that touches code outside the purview of the task
+ *   at hand. It could be handled as a tech debt item. */
+// Statuses shared by CMS and States
+const commonStatuses = ["Approved", "Disapproved", "Package Withdrawn"];
+// Statuses exclusive to CMS
+const commonCMSStatuses = [
+  ...commonStatuses,
+  "Pending",
+  "Pending - RAI",
+  "Pending - Approval",
+  "Pending - Concurrence",
+  "Submitted - Intake Needed",
```

Comment:
There was a requirement for state users to not see the CMS statuses - which is why they are converted in app-api.  While bringing the status lookup up into the UI exposes the CMS statuses, it at least does not show when a package is in one of the ones States aren't supposed to see.
I think the filter was data-driven, so only worked with statuses that were in the data.  Not sure if there is a possibility to do that here...

Reply:
I don't follow -- I've tested as a state user and none of the CMS statuses are visible. This isn't being used for anything except filter adding/removing logic that lets us show which values of `type` and `status` are hidden instead of which are applied ([core function](https://github.com/Enterprise-CMCS/macpro-onemac/pull/1309/files#diff-10c511453a78a20372c19e5174d4c393a67df6b7f85a5c56714b8c540a9376dcR117-R126), [application](https://github.com/Enterprise-CMCS/macpro-onemac/pull/1309/files#diff-10c511453a78a20372c19e5174d4c393a67df6b7f85a5c56714b8c540a9376dcR211-R220)). I do agree it's a redundancy, so I left a TODO about how we could move the mappings of those filter names to the `cmscommonlib` from where we actually map them in the api directory.

Reply:
I did find a bug, however, related to this. Not all the 1915 waiver types are represented in the array of types right now, so some chips do not appear when deselecting the waiver type from filters to hide it. 😂 Something for me to fix today!

Reply:
cmscommonlib is visible as well - I am aware this is one of my tedious spent-too-much-time-fighting-web-crawlers idiosyncrasies, AND I'm also aware we break this all the time.  I feel compelled to mention it when I see it, though.
<img width="1440" alt="Screenshot 2023-07-24 at 9 31 14 AM" src="https://github.com/Enterprise-CMCS/macpro-onemac/assets/70273267/12e2a2c1-5e06-46e4-84cd-9a5358be80a0">